packages/tokens/src/bin/tests/assets/expected-js-cunningham-tokens.css

```diff
@@ -5,4 +5,5 @@
 	--c--theme--font--weights--medium: 400;
 	--c--theme--font--families--base: Roboto;
 	--c--theme--spacings--s: 1rem;
-}
\ No newline at end of file
+	--c--theme--transitions--ease: linear;
```

Comment:
Maybe I'm too rigid but the transition variable must be included in the last commit. WDYT? 😅

Reply:
Indeed !